easybuild/easyconfigs/c/Clang/Clang-10.0.0-GCCcore-8.3.0.eb

```diff
@@ -43,27 +44,31 @@ checksums = [
     'b9a0d7c576eeef05bc06d6e954938a01c5396cee1d1e985891e0b1cf16e3d708',  # lld-10.0.0.src.tar.xz
     '270f8a3f176f1981b0f6ab8aa556720988872ec2b48ed3b605d0ced8d09156c7',  # libcxx-10.0.0.src.tar.xz
     'e71bac75a88c9dde455ad3f2a2b449bf745eafd41d2d8432253b2964e0ca14e1',  # libcxxabi-10.0.0.src.tar.xz
+    'acdf8cf6574b40e6b1dabc93e76debb84a9feb6f22970126b04d4ba18b92911c',  # clang-tools-extra-10.0.0.tar.xz
     'a424a9eb7f377b4f01d8c9b27fdd78e6a51a53819b263d2ca6d40a0e2368a330',  # libcxx-10.0.0-ppc64le.patch
 ]
 
 dependencies = [
     # since Clang is a compiler, binutils is a runtime dependency too
     ('binutils', '2.32'),
+    ('hwloc', '2.0.3'),
```

Comment:
@geimer We'll need to stick to hwloc 1.11.12 here (see failing tests)

Reply:
@boegel Let me double-check whether the OpenMP runtime also builds with hwloc 1.x. Stay tuned...

BTW: Why does see https://github.com/easybuilders/easybuild-easyconfigs/pull/12013#issuecomment-761571648 report
> List of changed easyconfig files in this PR: Boost-1.74.0-GCC-10.2.0.eb
> Boost-1.74.0-iccifort-2020.4.304.eb

???  Boost hasn't been touched in this PR...

Reply:
@boegel The build looks good to me even with hwloc 1.x.